lexers/o/objectivec.go

```diff
@@ -162,4 +178,22 @@ var ObjectiveC = internal.Register(MustNewLexer(
 			{`.*?\n`, Comment, nil},
 		},
 	},
-))
+).SetAnalyser(func(text string) float32 {
+	if objectiveCAnalyserKeywordsRe.MatchString(text) {
+		return 1.0
+	}
+
+	if strings.Contains(text, "@\"") {
```

Comment:
Let's use backticks here as well.
```suggestion
	if strings.Contains(text, `@`) {
```

Reply:
It's actually `if strings.Contains(text, `@"`) {`. Already fixed and pushed.